src/skydb.ts

```diff
@@ -49,9 +49,12 @@ export async function getJSON(
   }
 
   // Download the data in that Skylink.
-  // TODO: Replace with download request method.
   const skylink = entry.data;
-  const data = await this.getFileContent(skylink, opts);
+  const data = await this.getFileContent<Record<string, unknown>>(skylink, opts);
```

Comment:
Doesn't `getFileContent` return `Record<unknown>` now and not `Record<string, unknown>`?

Reply:
Thanks for the review. We had a discussion and decided to have it return generic data, since depending on the file it can return a string, json, or arraybuffer (e.g. binary files). We're going to also return the content-type header that can be examined.